llvm/tools/llvm-gsymutil/llvm-gsymutil.cpp

```diff
@@ -92,6 +92,7 @@ static uint64_t SegmentSize;
 static bool Quiet;
 static std::vector<uint64_t> LookupAddresses;
 static bool LookupAddressesFromStdin;
+static std::string SymbolTableFilename;
```

Comment:
Make optional?:
```
static std::optional<std::string> SymbolTableFilename;
```

Reply:
Just trying to keep it the same as other global string types (eg. OutputFilename is also an optional flag).
It will default to empty string if we didn't use that.